### PR TITLE
Added mailchimp link + removed formspree

### PR DESCRIPTION
**Pull request by abbey-titcomb**
Embedded the mailchimp sign up form so we can keep the current flow (subscribe -> /subscribe.html). Right now the form redirects to radicle.xyz/subscribed.html - hopefully this is the right link, if not lmk and I'll change it in mailchimp flow.

Removed some formspree code that I don't think is necessary for mailchimp, but check it out and let me know.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -40,12 +40,9 @@
         <h1 class="headline druk">Locally sourced.</h1>
         <div class="email-signup">
           <p><strong>A peer-to-peer stack for building software together. Sound fun?</strong></p>
-          <form class="main-cta" action="https://formspree.io/xrgbwygl" method="POST" autocomplete="off">
```
Comment (brandonhaslegs) [inline docs/index.html:43]: Can you keep in the `autocomplete="off"` bit in the form tag plz

File: docs/index.html [hunk 2]
```diff
@@ -298,12 +295,9 @@ <h1 class="druk">Stay up <span>to date</span></h1>
           <p>If you'd like to stay updated about product releases, project updates, and other announcements, hand us your e-mail and we'll keep you in the loop.</p>
           <br>
           <p>For updates, follow us on <a href="https://twitter.com/radicle_xyz" target="_blank"><span>Twitter</span></a>.</p>
-          <form action="https://formspree.io/xrgbwygl" autocomplete="off" method="POST">
```
Comment (brandonhaslegs) [inline docs/index.html:301]: Same here

**Review by brandonhaslegs — CHANGES_REQUESTED** [review]

File: docs/index.html [hunk 3]
```diff
@@ -40,12 +40,9 @@
         <h1 class="headline druk">Locally sourced.</h1>
         <div class="email-signup">
           <p><strong>A peer-to-peer stack for building software together. Sound fun?</strong></p>
-          <form class="main-cta" action="https://formspree.io/xrgbwygl" method="POST" autocomplete="off">
-            <input type="email" name="email" placeholder="Enter your email here" required />
+          <form class="main-cta" action="https://xyz.us2.list-manage.com/subscribe/post?u=6c7bba7ab992717860dcec6eb&amp;id=f960345a90" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
```
Comment (juliendonck) [inline docs/index.html:43]:
```suggestion
          <form class="main-cta" action="https://xyz.us2.list-manage.com/subscribe/post?u=6c7bba7ab992717860dcec6eb&amp;id=f960345a90" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate autocomplete="off">
```

File: docs/index.html [hunk 4]
```diff
@@ -298,12 +295,9 @@ <h1 class="druk">Stay up <span>to date</span></h1>
           <p>If you'd like to stay updated about product releases, project updates, and other announcements, hand us your e-mail and we'll keep you in the loop.</p>
           <br>
           <p>For updates, follow us on <a href="https://twitter.com/radicle_xyz" target="_blank"><span>Twitter</span></a>.</p>
-          <form action="https://formspree.io/xrgbwygl" autocomplete="off" method="POST">
-            <input type="email" name="email" placeholder="Enter your email here" required />
+          <form action="https://xyz.us2.list-manage.com/subscribe/post?u=6c7bba7ab992717860dcec6eb&amp;id=f960345a90" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
```
Comment (juliendonck) [inline docs/index.html:298]:
```suggestion
          <form action="https://xyz.us2.list-manage.com/subscribe/post?u=6c7bba7ab992717860dcec6eb&amp;id=f960345a90" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate autocomplete="off">
```

**Review by juliendonck — APPROVED** [review]
Looks good. One thing though, the logo you used on the mailchimp email is really bad quality. It's worth uploading a better version:

Screenshot 2020-11-13 at 15 03 08

File: docs/index.html (hunk 3 above)
Comment (abbey-titcomb) [reply docs/index.html]: resolved this in my latest commit

File: docs/index.html (hunk 2 above)
Comment (abbey-titcomb) [reply docs/index.html:301]: added

File: docs/index.html (hunk 1 above)
Comment (abbey-titcomb) [reply docs/index.html:43]: added

**Review by juliendonck — APPROVED** [review]

**Review by brandonhaslegs — CHANGES_REQUESTED** [review]
Why are there two different fonts in the MailChimp email? All Helvetica should work fine.

image

**Comment by brandonhaslegs** [comment]
And maybe should have a more generic email than your email in there?

**Comment by abbey-titcomb** [comment]
> Why are there two different fonts in the MailChimp email? All Helvetica should work fine.
> 
> image

Didn't notice. Can change in Mailchimp. 

Also will add a Radicle Foundation email when we decide which one to use. Ange is going to let me know.

**Review by brandonhaslegs — APPROVED** [review]

**Comment by abbey-titcomb** [comment]
Merging 👽

**Pull request merged**